extension/src/help/webHelpAbstraction.json

```diff
@@ -1,4 +1,5 @@
 {
+  "year": "2021",
```

Comment:
So now this is a hardcode year now?

Reply:
Not entirely,
It now represents "what was available" and theoretically "when" the webHelpAbstraction.json was last updated. I did build in a "fallback" of 2021 in the C#.